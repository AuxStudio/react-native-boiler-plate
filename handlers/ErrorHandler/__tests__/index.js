import React from 'react';
import renderer from 'react-test-renderer';
import { View } from 'react-native';

import { ErrorHandler } from '../';

it('renders a ErrorHandler with all props', () => {
  expect(
    renderer.create(
      <ErrorHandler dispatch={jest.fn()} uid="xxxx-xxxx-xxxx-xxxx">
        <View />
      </ErrorHandler>,
    ),
  ).toMatchSnapshot();
});

// TODO: Integration: test componentDidCatch

it('renders a ErrorHandler with minimum required props', () => {
  expect(
    renderer.create(
      <ErrorHandler dispatch={jest.fn()}>
        <View />
      </ErrorHandler>,
    ),
  ).toMatchSnapshot();
});
