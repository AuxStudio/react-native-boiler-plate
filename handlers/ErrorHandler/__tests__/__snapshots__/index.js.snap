// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a ErrorHandler with all props 1`] = `<View />`;

exports[`renders a ErrorHandler with minimum required props 1`] = `<View />`;
